GalaxyZooTouchTable/GalaxyZooTouchTable/Models/TableSubject.cs

```diff
@@ -0,0 +1,25 @@
+﻿using PanoptesNetClient.Models;
+using System;
+
+namespace GalaxyZooTouchTable.Models
+{
+    public class TableSubject
+    {
+        public string X { get; set; }
+        public string Y { get; set; }
+        public double RA { get; set; }
+        public double DEC { get; set; }
```

Comment:
Can you choose better names for `RA` and `DEC`? It's sort of confusing what they are

Reply:
Those are pretty standard names for astrometry, but I can do RightAscension and Declination

Reply:
Even those may be standard names for astrometry, the people working on the code may not be familiar with the terms. So I think it's best to be verbose with language here.